File: C++ Programs/evenOddWithoutMod.cpp
/* CPP PROGRAM TO CHECK WHETHER THE NUMBER IS EVEN OR ODD WITHOUT USING MOD% OPERATOR */
#include<iostream>
using namespace std;
int main()
{
	int num;
	cout<<"Enter Number=";
	cin>>num;
	if((num/2)*2==num)
	{
		cout<<"\n\n"<<num<<" Is An Even Number";
	}
	else
	{
		cout<<"\n\n"<<num<<" Is An Odd Number";
	}
	return 0;
}

File: Java Programs/evenOddWithoutMod.java
/* JAVA PROGRAM TO CHECK WHETHER THE NUMBER IS EVEN OR ODD WITHOUT USING MOD% OPERATOR */
import java.util.*;
public class evenOddWithoutMod
{
	public static void main(String[] args) 
	{
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter Number=");
		int num=sc.nextInt();
		if(((num/2)*2)==num)
		{
			System.out.println(+num+" Is An Even Number");
		}
		else
		{
			System.out.println(+num+" Is An Odd Number");
		}
	}
}